.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,36 +1,38 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a bug report to help us improve
 title: ""
 labels: ["bug"]
 assignees: ''
-
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+#### Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+#### To Reproduce
+<!-- Steps that can be taken to reproduce the behaviour -->
 
-**To Reproduce**
-Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+#### Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
```

Comment:
What is the value of commenting these lines out instead of removing them?
I don't have a strong opinion however on keeping or removing these descriptions. :)

Reply:
Yeah I guess I should have been more clear in the description.

The main benefit would be when someone doesn't fill out a section (say, screenshots or logs) and just ignores it (i.e. doesn't manually remove all the text). With comments, it won't have a bunch of placeholder text in it when they submit, the section would just be empty.

I don't have a strong opinion either, it was just something I thought might help as I was quickly cleaning up the markdown.